Core/Utilities/test/Test_Encode.py

```diff
@@ -278,6 +278,7 @@ def test_missingAttrToSerialize():
 
 
 @given(data=nestedStrategyJson)
+@settings(suppress_health_check=(HealthCheck.too_slow,))
```

Comment:
I am very surprised that this would trigger a timeout ! We also do not see it in travis. Are you sure it is necessary, and if yes, why ?

Reply:
Yes, my last RP did not pass travis tests through this time-out, the current changes to resolve this

Reply:
But this is very strange, it should not take al ot of time, and it was not before. Instead of hiding it, can you please investigate why it fails only on your PR ?

Reply:
trevis tests now pass without exclusion, so this change was canceled